Source/WebGPU/WebGPU/Device.mm

```diff
@@ -317,7 +317,7 @@ static uint32_t computeAppleGPUFamily(id<MTLDevice> device)
         ALLOW_DEPRECATED_DECLARATIONS_END
         id<MTLLibrary> library = [device newLibraryWithSource:@"[[vertex]] float4 vsNop() { return (float4)0; }" options:options error:&error];
         if (error)
-            WTFLogAlways("%@", error); // NOLINT
+            WTFNSLogAlways("%@", error); // NOLINT
```

Comment:
I wonder if we should just log the localized description of the error as a string here, instead of logging the object.

Reply:
That would likely be better, though realistically we could also release assert here as `[[vertex]] float4 vsNop() { return (float4)0; }` should always compile

Reply:
Will fix